shared_workflow/install_shared.py

```diff
@@ -215,7 +270,57 @@ def install_simulation(
             else:
                 sim_params_dict.update({key: value})
 
-    return root_params_dict, fault_params_dict, sim_params_dict, vm_params_dict
+    return sim_params_dict
+
+
+def generate_fault_params(root_folder, vel_mod_dir, fd_statcords, fd_statlist):
+    return {
+        FaultParams.root_yaml_path.value: sim_struct.get_root_yaml_path(
+            sim_struct.get_runs_dir(root_folder)
+        ),
+        FaultParams.vel_mod_dir.value: vel_mod_dir,
+        FaultParams.stat_coords.value: fd_statcords,
+        FaultParams.FD_STATLIST.value: fd_statlist,
+    }
+
+
+def generate_root_params(
+    root_params_base_dict, root_folder, extended_period, seed, stat_file_path, version
+):
+    root_params_base_dict["mgmt_db_location"] = root_folder
+    root_params_base_dict[RootParams.extended_period.value] = extended_period
+    root_params_base_dict[RootParams.version.value] = version
+    root_params_base_dict[RootParams.stat_file.value] = stat_file_path
+    root_params_base_dict[RootParams.stat_vs_est.value] = stat_file_path.replace(
+        ".ll", ".vs30"
+    )
+    root_params_base_dict[RootParams.stat_vs_ref.value] = stat_file_path.replace(
```

Comment:
@shot0829 Do we need this one?

Reply:
Its not being used anymore. And there is a way to emulate the same feature by specifying a value of lfvsref under bb. So i would say its good to be removed.